vllm/utils.py

```diff
@@ -223,15 +242,32 @@ def get(self, key: _K, default: Optional[_V] = None) -> Optional[_V]:
         if key in self.cache:
             value = self.cache[key]
             self.cache.move_to_end(key)
+
+            self._hits += 1
         else:
             value = default
+
+        self._total += 1
         return value
 
     def put(self, key: _K, value: _V) -> None:
         self.cache[key] = value
         self.cache.move_to_end(key)
         self._remove_old_if_needed()
 
+    def get_or_put(self, key: _K, default_factory: Callable[[], _V]) -> _V:
+        if key in self.cache:
+            value = self.cache[key]
+            self.cache.move_to_end(key)
+
+            self._hits += 1
+        else:
+            value = default_factory()
+            self.put(key, value)
+
+        self._total += 1
+        return value
```

Comment:
Where is this used?

Reply:
Oh, this is from #11341. Let me remove this.